internal/dataplane/kongstate/kongstate.go

```diff
@@ -94,6 +103,7 @@ func (ks *KongState) FillConsumersAndCredentials(log logrus.FieldLogger, s store
 				if k == "hash_secret" {
 					boolVal, err := strconv.ParseBool(string(v))
 					if err != nil {
+						// REVIEW: this is not an error blocking translate the consumer to Kong's consumer, do we add a translation error here?
```

Comment:
I would say that we should treat it as a translation error as well. Despite the fact we'll still attempt to configure the consumer when parsing of `hash_secret` fails, we're defaulting to `false` which might not be something the user desires. IMO it's worth bubbling up as an event.

Reply:
Added translation failure here.